### ISSUE TITLE
Won't compile in latest version of Rack yet

### ISSUE THREAD

**Issue by Strum** [antoniograzioli/Autodafe#1]
Just cloned this and it looks like at least the foldback.cpp file has yet to be changed to the new way of dealing with inputs params and outputs.

**Comment by antoniograzioli** [comment]
yes, this is for Version 0.32 as stated in the description. 
I already have reworked everything for the new version but haven't released yet on github

**Comment by antoniograzioli** [comment]
![commit032](https://user-images.githubusercontent.com/16060683/31090964-c4cf5860-a7a9-11e7-908b-1ca9a35d4123.png)

**Comment by Strum** [comment]
no worries.

Any idea when you will upload the new code?

I'm using the LFO a lot and you fixed the gates blurring into each other issue with the fundamental seq-3 so I've been using your seq-8 as well.nice work.

Your wave foldback module got me thinking as well and lead me to making a bunch of modules using similar principles including a clpiper, and the noise gate and envelope follower i'm working on right now.

Thanks again.

**Comment by antoniograzioli** [comment]
actually , no, I just updated everything this morning and I need to make sure everythin is OK.

Also, I would love to keep separate versions for v.0.32 and for v.0.40, time permitting, but VCVis still too young ad we will probably see lots of changes again..so I better stick with the "last" official version..

**Comment by IohannRabeson** [comment]
Hi, maybe you could keep v.0.40 changes separated using branches?

**Comment by Strum** [comment]
I was also going to suggest creating a development branch

**Comment by antoniograzioli** [comment]
Yes, as soon as I finish removing bugs from the latest 0.40 version (still some issues due to the new submenu systems and stuff like that)

in the meantime the Stable code for v. 0.32 is in this release:
https://github.com/antoniograzioli/Autodafe/releases/tag/Rack0.32

**Comment by Strum** [comment]
Cheers works now.
